b/bugsnag/bugsnag_ubi8.4.sh

```diff
@@ -0,0 +1,63 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: bugsnag
+# Version	: 6.20.0
+# Source repo	: https://github.com/bugsnag/bugsnag-ruby
+# Tested on	: UBI 8.4
+# Language      : Ruby
+# Travis-Check  : False
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sapana Khemkar {Sapana.khemkar@ibm.com}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e 
+
+PACKAGE_NAME=bugsnag
+PACKAGE_VERSION=v6.20.0
+PACKAGE_URL=https://github.com/bugsnag/bugsnag-ruby
+
+yum install -y nodejs nodejs-devel nodejs-packaging git wget curl ruby ruby-devel rubygem-rake procps libcurl-devel libffi-devel sqlite sqlite-devel
+
+yum install -y --allowerasing gcc gcc-c++ yum-utils make automake autoconf libtool gdb* binutils rpm-build gettext wget
+
+yum-config-manager --add-repo http://mirror.centos.org/centos/8/AppStream/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/PowerTools/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/virt/ppc64le/ovirt-44/
+
+wget https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official && mv RPM-GPG-KEY-CentOS-Official /etc/pki/rpm-gpg/. && rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
```

Comment:
I am not sure this will work now and in any case we need to remove unwanted repos and packages